### PR TITLE
corrected storage terminology, specific location for HDD can be speci…

### PR DESCRIPTION
**Pull request by vaibhav-rbs**
…fied.

### CODE REVIEW

**Comment by vaibhav-rbs** [comment]
rebuild

File: src/app/pages/vm/vm-wizard/vm-wizard.component.ts [hunk 1]
```diff
@@ -156,18 +156,20 @@ export class VMWizardComponent {
           isHidden: false
         },
         {
-          type: 'select',
+          type: 'explorer',
           name: 'datastore',
-          placeholder : T('Select a datastore'),
-          tooltip: T('Choose a datastore for the new zvol.'),
+          placeholder : T('Select a zpool/datastore'),
```
Comment (wblock) [inline src/app/pages/vm/vm-wizard/vm-wizard.component.ts]: As noted in the other bug, "datastore" is a VMware term. In ZFS, they are datasets.  Additionally, "zpool" is redundant with just "pool".

So this should be just T('Select a pool or datastore')

File: src/app/pages/vm/vm-wizard/vm-wizard.component.ts [hunk 2]
```diff
@@ -156,18 +156,20 @@ export class VMWizardComponent {
...
+          tooltip: T('Choose a zpool/datastore for the new zvol.'),
```
Comment (wblock) [inline src/app/pages/vm/vm-wizard/vm-wizard.component.ts]: As above, 'Choose a pool or dataset for the new zvol.'

File: src/app/pages/vm/vm-wizard/vm-wizard.component.ts [hunk 3]
```diff
@@ -156,18 +156,20 @@ export class VMWizardComponent {
...
+          isHidden: false,
+          initial: '/mnt',
+          explorerType: 'directory'
         },
         {
           type: 'explorer',
           name: 'hdd_path',
           placeholder: T('Select an existing disk'),
-          tooltip: T('Browse to the desired datastore on the disk.'),
+          tooltip: T('Browse to the desired zpool/datastore on the disk.'),
```
Comment (wblock) [inline src/app/pages/vm/vm-wizard/vm-wizard.component.ts]: As above: 'Browse to the desired pool or dataset.'

**Pull request merged**